metadata-transformer/src/main/java/no/unit/nva/metadata/service/MetadataService.java

```diff
@@ -138,13 +139,24 @@ private Model normalizeStatements(RepositoryResult<Statement> statements) throws
                 model.add(toDctermsDate(statement));
             } else if (isHighWireLanguage(statement)) {
                 model.add(toDctermsLanguage(statement));
+            } else if (isHighWireAuthor(statement)) {
```

Comment:
I'm thinking maybe we should, at some point, refactor to not use the term "highwire" in our code, since the connection between "highwire" and "citation_" is not immediately obvious

Reply:
You're right, it isn't…I suppose the real fix is to define "HighWire", but I think that that too may as well be a matter of defining the tags we understand exist and accept in the "citation_" 'pseudonamespace'.